AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -600,6 +600,8 @@ hgtv.com#@#.showads
 !
 !
 !
+otomobilgunluklerim.com#$#.ad_block_detected { overflow: auto !important; }
+otomobilgunluklerim.com##.adblockalert
```

Comment:
You'll break page scrolling on cb platforms.
In this case, you should use
```
otomobilgunluklerim.com#$#body.ad_block_detected { overflow: auto !important; }
otomobilgunluklerim.com#$#.adblockalert { display: none !important; }
```
Btw, don't forget about cb platforms. Added a rule for them directly.

Reply:
Thanks. https://github.com/AdguardTeam/AdguardFilters/pull/103609/commits/371af0d7b7f20d014cd8de9df74a0bb694184fb5